reports/Colorado.md

```diff
@@ -6,15 +6,16 @@ On June 27th, citizens of Aurora held a violin vigil in honor of Elijah McClain.
 
 Aurora police arrived in riot gear to disperse the vigil. When participants refused, riot police beat them with batons and pepper-sprayed them.
 
-tags: strike, beat, baton, pepper-spray, spray
+tags: strike, beat, baton, pepper-spray, spray, tear-gas
```

Comment:
I'm not sure about the tear gas. It may have just been smoke grenades, so unless there are recovered canisters or something, I would prefer not to make the assumption

Reply:
That's a good point. Aurora PD said they **didn't** use tear-gas, but that smoking canister being tossed is suspicious.

Reply:
Yeah but it could easily just be a smoke grenade. So I think you could include the ambiguity in the description, but I would remove the tag.

Reply:
removed the tag